doc/quick-start.rst

```diff
@@ -173,6 +173,12 @@ this ``dune`` file:
      (c_names         mystubs)
      (c_flags         (-I/blah/include))
      (c_library_flags (-lblah)))
+     
+And compile everything using 
+
+.. code:: bash
+
+    dune build libmylib_stubs.a mylib.cmxa
```

Comment:
`dune build` is enough here since the library has a public_name, any particular reason to refer to the artifacts directly?